__tests__/fixtures/workflow/action_docs_workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: "Action update docs action"
```

Comment:
This is just a normal action.yml? I was expecting here workflows

Reply:
Ok to quick look, was confused by the branding section.

Reply:
Corrected the syntax of this file to be a workflow not an action